docs/GettingStarted.md

```diff
@@ -228,7 +228,7 @@ What went wrong here? Pine.js is simply preventing us from violating the constra
 To correctly modify the device we can try:
 
 ```
-$ curl -X PUT -d name=testdevice -d note=updatednote -d type=raspberry http://localhost:1337/example/device(1)
+$ curl -X PUT -d name=testdevice -d note=updatednote -d type=raspberry "http://localhost:1337/example/device(1)"
```

Comment:
In general I'd use single-quotes when possible because it also helps rule out issues with eg `$filter` being evaluated as an env var but for these cases it doesn't matter